src/OrderProvider.cs

```diff
@@ -155,15 +156,27 @@ public override Schema GetOriginalSourceSchema()
                     tablesToRemove.Add(table);
                 else if (table.Name == "EcomOrders")
                 {
-                    table.AddColumn(new SqlColumn(("OrderCustomerAccessUserExternalId"), typeof(string), SqlDbType.NVarChar, table, -1,
-                                                  false, false, true));
+                    table.AddColumn(new SqlColumn(SourceColumnNameForDestinationOrderCustomerAccessUserId, typeof(string), SqlDbType.NVarChar, table, -1, false, false, true));
```

Comment:
Is the use of SourceColumnNameForDestinationOrderCustomerAccessUserId really right where it's been used here?
I thought OrderCustomerAccessUserExternalId was made as the constant for "OrderCustomerAccessUserExternalId"